contracts/SecurityTokenRegistry.sol

```diff
@@ -164,9 +164,15 @@ contract SecurityTokenRegistry is ISecurityTokenRegistry, EternalStorage {
      * @param _polyToken is the address of the POLY ERC20 token
      * @param _owner is the owner of the STR
      */
-    function initialize(address _polymathRegistry, address _STFactory, uint256 _stLaunchFee, uint256 _tickerRegFee, address _polyToken, address _owner) payable external {
+    function initialize(
+        address _polymathRegistry, address _STFactory, uint256 _stLaunchFee, uint256 _tickerRegFee, address _polyToken, address _owner)
```

Comment:
If splitting function life this, all variables must go in separate lines
Look at https://solidity.readthedocs.io/en/v0.4.25/style-guide.html for the style guide